README.md

```diff
@@ -216,6 +216,87 @@ stripe.setAppInfo({
 
 This information is passed along when the library makes calls to the Stripe API.
 
+### Auto-pagination
+
+As of stripe-node 6.11.0, you may auto-paginate list api methods.
+We provide a few different API's for this to aid with a variety of node versions and styles.
```

Comment:
Sorry the nit, but I think we usually use "APIs" (see [examples from Google](https://www.google.ca/search?q=apis+site%3Astripe.com&oq=apis+site%3Astripe.com&aqs=chrome..69i57.2805j0j7&sourceid=chrome&ie=UTF-8)). There's a little disagreement, but without an apostrophe is usually considered the right way to pluralize an acronym.

Reply:
nice, yes, thanks!